README.md

```diff
@@ -22,13 +22,24 @@ I occasionally tweet my pursuits (sometimes in Slovenian) on [@TitPetric](https:
 
 # Using
 
+## Docker CLI
+
 ```
 docker run -d --cap-add SYS_PTRACE \
            -v /proc:/host/proc:ro \
            -v /sys:/host/sys:ro \
            -p 19999:19999 titpetric/netdata
 ```
 
+## Docker Compose
+
+Use the sample [docker-compose.yml](docker-compose.yml) file.
+
+```
+mkdir netdata && cd netdata && wget https://raw.githubusercontent.com/titpetric/netdata/master/docker-compose.yml
+docker-compose up --build -d
```

Comment:
There is no Dockerfile specified in the compose file. I think the --build can be removed.

Reply:
I'm not sure that `docker-compose` is gonna pull the new version of image every time without `--build`, but if I'm ready to believe you and remove the flag. Anyway it's just a reference, and the image will absolutely start without `--build`

Reply:
``docker-compose Pulli ist needed to update all images in a compose file.